autoptsclient_common.py

```diff
@@ -1221,7 +1228,7 @@ def check_args(self, args):
             board_power(args.ykush, True)
             time.sleep(1)
 
-        if 'tty_file' in args:
+        if args.tty_file:
```

Comment:
It looks more like a bug before. When we use QEMU, we will report an error(Nonetype)

Reply:
Oh, I came here too late and I made you some conflict here with https://github.com/intel/auto-pts/pull/479.

Reply:
It was truly a bug. Just 'if args.tty_file:' wasn't enough though. tty_file is not always in that args.

Reply:
I will rebase...

Reply:
@mkasenberg Updated.